tests/src/test/resources/files/crac/epic15/jsonCrac_ep15us11-3case1.json

```diff
@@ -1,160 +1,218 @@
+        400.0
+      ],
+      "thresholds": [
+        {
+          "unit": "ampere",
+          "min": -433.0,
+          "max": 433.0,
+          "side": "left"
+        }
+      ]
+    },
+    {
+      "id": "be4_fr5_co1 - BBE4AA11->FFR5AA11  - co1_be1_fr5 - curative",
+      "name": "be4_fr5_co1",
+      "networkElementId": "BBE4AA11 FFR5AA11 1",
+      "operator": null,
+      "instant": "curative",
+      "contingencyId": "co1_be1_fr5",
+      "optimized": true,
+      "monitored": false,
+      "reliabilityMargin": 0.0,
+      "iMax": [
+        2000.0
+      ],
+      "nominalV": [
+        400.0
+      ],
+      "thresholds": [
+        {
+          "unit": "ampere",
+          "min": -433.0,
+          "max": 433.0,
+          "side": "left"
+        }
+      ]
+    },
+    {
+      "id": "be4_fr5_co1 - BBE4AA11->FFR5AA11  - co1_be1_fr5 - outage",
+      "name": "be4_fr5_co1",
+      "networkElementId": "BBE4AA11 FFR5AA11 1",
+      "operator": null,
+      "instant": "outage",
+      "contingencyId": "co1_be1_fr5",
+      "optimized": true,
+      "monitored": false,
+      "reliabilityMargin": 0.0,
+      "iMax": [
+        2000.0
+      ],
+      "nominalV": [
+        400.0
+      ],
+      "thresholds": [
+        {
+          "unit": "ampere",
+          "min": -9999.0,
+          "max": 9999.0,
+          "side": "left"
+        }
+      ]
+    },
+    {
+      "id": "be4_fr_5_n - BBE4AA11->FFR5AA11 - preventive",
+      "name": "be4_fr_5_n",
+      "networkElementId": "BBE4AA11 FFR5AA11 1",
+      "operator": null,
+      "instant": "preventive",
+      "optimized": true,
+      "monitored": false,
+      "reliabilityMargin": 0.0,
+      "iMax": [
+        2000.0
+      ],
+      "nominalV": [
+        400.0
+      ],
+      "thresholds": [
+        {
+          "unit": "ampere",
+          "max": 9999.0,
+          "side": "left"
+        }
+      ]
+    }
+  ],
+  "angleCnecs": [],
+  "voltageCnecs": [],
+  "pstRangeActions": [],
+  "hvdcRangeActions": [
+    {
+      "id": "ARA_HVDC",
+      "name": "ARA_HVDC",
+      "operator": "FR",
+      "initialSetpoint": 0.0,
+      "ranges": [
+        {
+          "min": -2000.0,
+          "max": 2000.0
+        }
+      ],
+      "networkElementId": "BBE2AA11 FFR3AA11 1",
+      "speed": 1,
+      "onConstraintUsageRules": [
+        {
+          "instant": "auto",
+          "usageMethod": "forced",
+          "cnecId": "be4_fr5_co1 - BBE4AA11->FFR5AA11  - co1_be1_fr5 - auto"
+        }
+      ]
+    },
+    {
+      "id": "CRA_HVDC",
+      "name": "CRA_HVDC",
+      "operator": "FR",
+      "initialSetpoint": 0.0,
+      "ranges": [
+        {
+          "min": -2000.0,
+          "max": 2000.0
+        }
+      ],
+      "networkElementId": "BBE2AA11 FFR3AA11 1",
+      "onConstraintUsageRules": [
+        {
+          "instant": "curative",
+          "usageMethod": "available",
+          "cnecId": "be4_fr5_co1 - BBE4AA11->FFR5AA11  - co1_be1_fr5 - curative"
```

Comment:
General remark that I thought of reading this.
Are we gonna have issues with CNECs having the same Ids but one being flowCnec and the other angleCnec ? Because with this PR we're not going to be able to know which CNEC it refers to as we lost the information in the RA.

Reply:
I checked this and it is impossible to create two CNECs with the same id even if they are of different types so we are good

Reply:
So how are we going to name two cnecs that are exactly the same but one is a FlowCnec and the other one an AngleCnec ?

Reply:
So far we cannot and the easiest way is to simply add "- FLOW" or "- ANGLE" at the end of the CNEC's name to differentiate which is which

Imo it is not such a problem because the network elements for the different types of CNECs are slightly different so it make sense to me if the names are different too